cpp/arcticdb/version/local_versioned_engine.hpp

```diff
@@ -57,6 +57,16 @@ struct SortMergeOptions {
     bool prune_previous_versions_;
 };
 
+folly::Future<folly::Unit> delete_trees_responsibly(
```

Comment:
Why did this move?

Reply:
For https://github.com/man-group/arcticdb-enterprise/pull/133